cloud-openstack-server/src/main/java/jetbrains/buildServer/clouds/openstack/OpenstackCloudImage.java

```diff
@@ -1,24 +1,23 @@
 package jetbrains.buildServer.clouds.openstack;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
-
-import org.jclouds.openstack.nova.v2_0.features.ServerApi;
-import org.jclouds.openstack.nova.v2_0.options.CreateServerOptions;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
 import com.jcabi.log.VerboseRunnable;
-
 import jetbrains.buildServer.clouds.CloudErrorInfo;
 import jetbrains.buildServer.clouds.CloudImage;
 import jetbrains.buildServer.clouds.CloudInstanceUserData;
 import jetbrains.buildServer.clouds.InstanceStatus;
+import jetbrains.buildServer.log.Loggers;
```

Comment:
As simple collaborator and not admin on this repo, I can't link SonarCloud Quality Gate in PR, but see [result](https://sonarcloud.io/dashboard?id=jetbrains.buildServer.clouds%3Acloud-openstack&pullRequest=44), this import is useless.
The most simple way is perhaps to use the file of two previous commit.